pkg/cloudprovider/suite_test.go

```diff
@@ -674,6 +703,8 @@ var _ = Describe("CloudProvider", func() {
 			awsEnv.EC2API.DescribeSecurityGroupsOutput.Set(&ec2.DescribeSecurityGroupsOutput{SecurityGroups: []*ec2.SecurityGroup{}})
 			// Instance is a reference to what we return in the GetInstances call
 			instance.SecurityGroups = []*ec2.GroupIdentifier{{GroupId: aws.String(fake.SecurityGroupID())}}
+			awsEnv.SecurityGroupCache.Flush()
```

Comment:
Should we just add this as a beforeEach()?

Reply:
No, we need to flush out the cache for this test

Reply:
We already do that